src/SFA.DAS.EAS.Web.IntegrationTests/TestUtils/ApiTester/ApiIntegrationTester.cs

```diff
@@ -89,22 +82,13 @@ public Task<CallResponse> InvokeGetAsync(CallRequirements call)
         }
 
         public Task<CallResponse<TResult>> InvokeGetAsync<TResult>(CallRequirements call)
-        {
-            return GetResponseAsync<TResult>(call);
-        }
-
-        /// <summary>
-        ///     Use the DI container used for the test to resolve the specified service.
-        /// </summary>
-        public T Resolve<T>()
         {
             EnsureStarted();
+            EnsureInstanceOfDbContextsHaveTransactions();
```

Comment:
why doesn't the existing ioc setup handle this?

Reply:
The existing IOC does not. But this is a good point (it should), I'll take a look.